packages/components/data-table/src/cell.js

```diff
@@ -196,15 +197,18 @@ DataCell.defaultProps = {
 
 const FooterCell = (props) => (
   <BaseFooterCell numberOfColumns={props.numberOfColumns}>
-    <CellInner alignment={props.alignment} isCondensed={props.isCondensed}>
+    <CellInner
+      isCondensed={props.isCondensed}
+      horizontalCellAlignment={props.horizontalCellAlignment}
+    >
       {props.children}
     </CellInner>
   </BaseFooterCell>
 );
 FooterCell.displayName = 'FooterCell';
 FooterCell.propTypes = {
   children: PropTypes.node.isRequired,
-  alignment: PropTypes.oneOf(['left', 'center', 'right']),
+  horizontalCellAlignment: PropTypes.oneOf(['left', 'center', 'right']),
```

Comment:
Are or shoudl we defaulting this?

Reply:
Yeah, forgot to add the default on the Footer.